include/swift/AST/DiagnosticEngine.h

```diff
@@ -113,6 +114,7 @@ namespace swift {
     TypeRepr,
     FullyQualifiedType,
     PatternKind,
+    DescriptivePatternKind,
```

Comment:
You should be able to remove `PatternKind` here and below, as `DescriptivePatternKind` should always be used for diagnostics.

Reply:
Oh you mean it's a complete replacement rather than an addition? (Just for this file in context?)

Reply:
Yes